pkgs/applications/blockchains/go-ethereum.nix

```diff
@@ -11,7 +11,21 @@ buildGoModule rec {
     sha256 = "1yqqflp73yvjy6bp05xd1nv5fc6p1nx7g4spbssxf3ws96pdh425";
   };
 
-  modSha256 = "07xrw3fivfpbkg4mp8ghrj1bishfas82dbd780fymgs2h74iigf3";
+  usb = fetchFromGitHub {
+    owner = "karalabe";
+    repo = "usb";
+    rev = "911d15fe12a9c411cf5d0dd5635231c759399bed";
+    sha256 = "0asd5fz2rhzkjmd8wjgmla5qmqyz4jaa6qf0n2ycia16jsck6wc2";
+  };
+
+  vendorSha256 = "12aw8b8ci7nssidh1yz8d2hk43csdacq5jca9y70rjakpl3zndsz";
+
+  overrideModAttrs = (_: {
+      postBuild = ''
+      cp -r --reflink=auto ${usb}/libusb vendor/github.com/karalabe/usb
+      cp -r --reflink=auto ${usb}/hidapi vendor/github.com/karalabe/usb
+      '';
+    });
```

Comment:
Just more formatting nitpicks, don't mind me...

This should probably look more like:

```
  overrideModAttrs = _: {
    postBuild = ''
      cp -r --reflink=auto ${usb}/libusb vendor/github.com/karalabe/usb
      cp -r --reflink=auto ${usb}/hidapi vendor/github.com/karalabe/usb
    '';
  };
```

(Applies to various files.)

Other indentation issues I noticed: https://github.com/NixOS/nixpkgs/pull/86376/files#diff-f1d73f2dfe1089118177778bd6b8e2b5, https://github.com/NixOS/nixpkgs/pull/86376/files#diff-d6d18e3aa6d1cc223b1b69d1d394dfa3, https://github.com/NixOS/nixpkgs/pull/86376/files#diff-775b8f1049046bc48a248e44668e74eb.

Reply:
Why not just send a pr?